src/utils.ts

```diff
@@ -1,6 +1,30 @@
+function base64ToUint8Array(b64: string): Uint8Array {
```

Comment:
this proposal will improve this in the future: https://github.com/tc39/proposal-arraybuffer-base64